### DISCUSSION TITLE
Signup Error Assistance

### DISCUSSION THREAD

**Discussion by bkuddin** [vercel/community#4480] [Account Issue]
### Question

I keep getting this error during the signup process for my Vercel account. The error message I received was 
"**We are unable to complete your signup. Please read this guide for further details.**"

Is there any way to make a Vercel account, and if so, what are the options? Note: I tried using GitHub and GitLab, but the result was the same.

**Comment by amyegan** [comment]
Hi @bkuddin. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. If so, please follow those steps. 🙏 

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

If you let me know your case number, I can double check that it has the right priority with the team that can help with this.

**Reply by amyegan** [reply]
In case you already had an older Vercel account, you could also try to [log in](https://vercel.com/login) instead of sign up. Please let me know if that works for you.
